frontend/app/views/tier/upgrade/paidToPaid.scala.html

```diff
@@ -20,7 +20,7 @@
           <form action="@routes.TierController.upgrade(summary.target.tier)" method="POST" class="js-form" id="payment-form">
               @CSRF.formField
 
-              @fragments.page.pageHeader("Great, glad to see you've decided to become a " + summary.target.tier.name, Some("Join as an annual Partner or Patron Member and get 2 months free"))
```

Comment:
@paulbrown1982 Don't suppose you can remember how this free period used to be applied?